deployments/data8/image/environment.yml

```diff
@@ -21,6 +22,7 @@ dependencies:
 
 # Items not in conda forge
 - pip:
+  - notebook==7.0.6
```

Comment:
@balajialg notebook should be put into the conda package section rather than under `pip` in my opinion. Version 7.0.6 is available in conda-forge, https://anaconda.org/conda-forge/notebook